pluma/pluma-commands-search.c

```diff
@@ -271,7 +280,9 @@ do_find (PlumaSearchDialog *dialog,
 	}
 
 	g_free (search_text);
-	
+	if(match_regex)
+		last_replace_text = g_strdup(pluma_search_dialog_get_replace_text (dialog));
```

Comment:
I think `last_replace_text` should be freed before copying a new string to it.

Reply:
@monsta fixed.